Source/WebCore/dom/MouseEvent.cpp

```diff
@@ -194,14 +195,14 @@ MouseButton MouseEvent::button() const
     return isKnownButton ? static_cast<MouseButton>(m_button) : MouseButton::Other;
 }
 
-int MouseEvent::which() const
+unsigned MouseEvent::which() const
 {
     // For the DOM, the return values for left, middle and right mouse buttons are 0, 1, 2, respectively.
     // For the Netscape "which" property, the return values for left, middle and right mouse buttons are 1, 2, 3, respectively.
     // So we must add 1.
     if (!m_buttonDown)
         return 0;
-    return m_button + 1;
+    return (unsigned)(m_button + 1);
```

Comment:
Please use static_cast<unsigned> instead.

Reply:
Done in [49d41ba](https://github.com/WebKit/WebKit/commit/49d41bae5e42c446c7b7eeab027bd5535a50824c).